book/07-git-tools/sections/reset.asc

```diff
@@ -237,11 +236,11 @@ So you can selectively unstage or revert content.
 
 ==== Squashing
 
-Let's look at how to do something interesting with this newfound power – squashing commits.
+Let's look at how to do something interesting with this newfound power -- squashing commits.
 
 Say you have a series of commits with messages like ``oops.'', ``WIP'' and ``forgot this file''.
 You can use `reset` to quickly and easily squash them into a single commit that makes you look really smart.
-(<<_squashing>> shows another way to do this, but in this example it's simpler to use `reset`.)
+(<<_squashing>> shows another way to do this but, in this example it's simpler to use `reset`.)
```

Comment:
I disagree with this change. If it were "clause, but, clause, clause" I agree that the comma should come before "but", but in a "clause, but clause" structure the comma should stay before.

Oh, I just found a source to back up my instinct:

> Rule: Use a comma before a coordinating conjunction that joins two independent clauses.
>> She purchased the car, but she declined the extended warranty.

Reply:
I stand corrected, and will fix.

Reply:
OK, fixed, commit amended and pushed.